pkg/python/packaging/parse.go

```diff
@@ -33,18 +33,31 @@ func (*Parser) Parse(r dio.ReadSeekerAt) ([]types.Library, []types.Dependency, e
 	if l := h.Get("License-Expression"); l != "" {
 		license = l
 	} else if l := h.Get("License"); l != "" {
-		license = l
+		// The license field can contain information about different licenses, license exceptions , etc.:
+		// https://packaging.python.org/en/latest/specifications/core-metadata/#license,
+		// but it is impossible to define a delimiter to separate them.
+		// Mark them, so we don't have to separate them later.
+		license = types.NonSeparableLicensePrefix + l
 	} else {
```

Comment:
I was thinking about checking `license classifiers` before `license`, but in this case we might miss some nuances (like license exclusion).
So I left the old order.